shipyard_blueprints/hubspot/shipyard_hubspot/hubspot_utils.py

```diff
@@ -159,28 +159,28 @@ def column_to_hubspot(
     :param column_type: The column type: HUBSPOT_OBJECT_ID, HUBSPOT_ALTERNATE_ID
     :return: importRequest Column Mapping
     """
-    if column_type:
```

Comment:
Made a change here to make the code slightly more efficient. Underlying logic should be unchanged